components/Common/AvatarGroup/index.tsx

```diff
@@ -0,0 +1,41 @@
+'use client';
+import { useState } from 'react';
+import styles from './index.module.css';
+import type { FC } from 'react';
+import Avatar from './avatar';
+
+type AvatarGroupProps = {
+  avatars: string[];
+  limit?: number;
+  forceShow?: boolean;
+};
+
+const AvatarGroup: FC<AvatarGroupProps> = ({
+  limit = 10,
+  avatars,
+  forceShow,
+}) => {
+  const [showMore, setShowMore] = useState(forceShow?.valueOf() || false);
+
+  const handleShowMoreClick = () => {
+    if (forceShow) return;
+    setShowMore(true);
+  };
+
+  return (
+    <div className={styles.avatarGroup}>
+      {showMore
+        ? avatars.map((avatar, index) => <Avatar key={index} src={avatar} />)
```

Comment:
This should be memoized and not done directly within the render of the method

Reply:
I don't have 100% understand but I think I have do what you propose 😅

Reply:
Pretty muuch do the map within a `useMemo`

Reply:
Ok i will test that !